HelpSource/Classes/SimpleNumber.schelp

```diff
@@ -974,52 +974,85 @@ stores this on the given stream
 method:: printOn
 prints this on the given stream
 
+
 subsection:: random
+Unless noted otherwise, when the receiver and arguments to these methods are all link::Classes/Integer::s, the return will be as well.
 
 method:: coin
 Let emphasis::x:: be the receiver clipped to the range [0, 1]. With probability emphasis::x::, return true. With probability 1 - emphasis::x::, return false.
+code::
+0.4.coin;
+
+{ SinOsc.ar(110).coin }.plot;
+::
 
 method:: rand
-returns:: Random number from zero up to the receiver, exclusive.
+Random number uniformly distributed in the range [0, this).
+code::
+10.rand;
+
+{ SinOsc.ar(110).rand }.plot;
+::
 
 method:: rand2
-returns:: a random number from -this to +this.
+Random number uniformly distributed in the range [-this, +this].
+code::
+10.rand2;
+
+{ SinOsc.ar(110).rand2 }.plot;
+::
 
 method:: rrand
-argument::aNumber
-the upper limit
+Random number uniformly distributed in the range [this, hi]. Note that unlike link::Classes/SimpleNumber#-rand::, the range is inclusive of the upper bound.
+argument::hi
+The upper limit of the range.
 argument::adverb
-returns:: a random number in the interval ]a, b[.
-discussion::
-If both a and b are link::Classes/Integer:: then the result will be an link::Classes/Integer::.
+discussion:: Equivalent to code::(hi - lo) * 1.0.rand + lo::, except for the inclusive upper bound.
 
 method:: linrand
-returns:: a linearly distributed random number from zero to this.
+Random number linearly distributed in the range [0, this], i.e., sampled from a distribution over [0, 1] with PDF 2(1 - x) and scaled by this.
```

Comment:
"PDF" will be hard for people to search on the internet :) Maybe expand?

Reply:
can do.